### ISSUE TITLE
How to flash to SD

### ISSUE THREAD

**Issue by TopperBG** [forrestbao/NanoPi#1]
> Once finished, you will find u-boot.bin, zImage, and rootfs.tar.gz in buildroot/output/images you can now flash these to your SD card.

How to flash it - in one boot partition, in two partition (one boot with u-boot.bin & zImage, second rootfs with excrated rootfs.tar.gz) or just place files in fat32? ext4? partition

Please advice
Thank you

**Comment by jrspruitt** [comment]
I believe what you need is this, it creates the SD Card with the bootloader
in the correct place, if you need to use your own, the should got in the
prebuilt directory, you can look at the fusing.sh script for what happens.

https://github.com/friendlyarm/sd-fuse_nanopi

**Comment by TopperBG** [comment]
Yeah, already use it, also my script (very similar) but have a bad feeling like I'm not change the u-boot or other content ...
Thank you, will continue to dig.

**Comment by jkent** [comment]
Hi TopperBG -- If you are using the NanoPi 1, you may want to try nanoboot:
https://github.com/ARMWorks/NanoPi-nanoboot

It uses a FAT filesystem for /boot and ext4 for /.  Its configurable with a
simple text file like the RasberryPi's bootloader.

Best regards,
Jeff Kent

**Comment by TopperBG** [comment]
I'm using NanoPi Air but there is no big difference.

**Comment by jkent** [comment]
There is a HUGE difference between the NanoPi and the NanoPi Neo Air.  The
NanoPi has a Samsung S3C2451 and the NanoPi Neo Air is an Allwinner H3.

These are the tools you need:
https://github.com/friendlyarm/h3_lichee/tree/master/fa_tools

**Comment by TopperBG** [comment]
According flash u-boot and kernel on SD card there is no difference.
Also I want to build Buildroot kernel not FA lichee with full size Ubuntu.
